src/database/KaiserInventoryDao.java

```diff
@@ -0,0 +1,69 @@
+package database;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+public class KaiserInventoryDao {
```

Comment:
fix the formatting of this class, you hav e a lot of tabs and new lines that are not needed